azurerm/provider.go

```diff
@@ -233,7 +233,11 @@ func Provider() terraform.ResourceProvider {
 			"azurerm_container_group":                                    resourceArmContainerGroup(),
 			"azurerm_container_registry":                                 resourceArmContainerRegistry(),
 			"azurerm_container_service":                                  resourceArmContainerService(),
-			"azurerm_cosmosdb_account":                                   resourceArmCosmosDBAccount(),
```

Comment:
I think we should rename this to `azurerm_cosmos_account`

Reply:
given the product naming appears to be `Cosmos DB Service` ([1](https://azure.microsoft.com/en-us/services/cosmos-db/) / [2](https://docs.microsoft.com/en-us/azure/cosmos-db/)) - I don't necessarily think it does make sense to ditch the `db` suffix here?